README.md

```diff
@@ -142,3 +140,9 @@ username               vault-edu
 ## Developing
 
 You can run `make dev-vault` in the root of the repo to start up a development vault server and automatically register a local build of the plugin. You will need to have a built `vault` binary available in your `$PATH` to do so.
+
+### Acceptance tests
+
+Run `make testacc`.
+
+For ARM-based macs, you need to run with `make testacc COUCHBASE_VERSION=7.1.3`.
```

Comment:
Could maybe update the tests to default to couchbase 7? It seems unlikely there will be breakages in 6 anyway now that 7 is the latest major.